openmc/tallies.py

```diff
@@ -897,10 +904,18 @@ def get_values(self, scores=[], filters=[], filter_bins=[],
 
                 # Add indices for each bin in this Filter to the list
                 for bin in bins:
-                    filter_indices[i].append(
-                        self.get_filter_index(filter.type, bin))
+                    if user_filter:
+                        filter_indices[i].append(
+                            self.get_filter_index(filter.type, bin))
+                    else:
+                        filter_indices[i].append(
+                            self.get_filter_index(filter.type, bin))
```

Comment:
Good point. They weren't the same at the in my initial implementation, but it appears they are now as I worked the algorithms out.